datasets/gov/vvkt/vaistiniai_preparatai.csv

```diff
@@ -0,0 +1,68 @@
+,datasets/gov/vvkt/vaistiniai_preparatai,,,,,,,,,,,,Valstybinės vaistų kontrolės registruotų vaistinių preparatų duomenų rinkinys,"""Rinkinyje pateikiami duomenys apie vaistinius preparatus sukauptus VVKT (Valstybinės vaistų kontrolės tarnybos). Rinkinyje nurodomos vaistinių preparatų charakteristikos: preparato pavadinimas, preparate esanti veiklioji medžiaga, stiprumas, vaistinio preparato farmacinė forma ir vartojimo būdai, kokiai grupei ir pogrupiui priskiriamas vaistinis preparatas, vaistinio preparato recepto poreikis, stebėjimo ir kontrolės būsenos, pirmos registracijos, bei paskutinio perregistravimo data, dabartinė preparato stadija, taip pat preparatą registruojančios įmonės pavadinimas ir šalis. Rinkinyje taip pat yra informacijos apie vaistinių preparatų pakuotes: pakuotės tiekimo būsena, pakuotės tipas, pakuotės aprašymas. Rinkinys sudarytas iš dviejų lentelių:
+* Vaistinis preparatas (Informacija apie vaistinį preparatą ir jo pakuotes) - viena eilutė yra informacija apie vaistinį preparatą ir jo pakuotę. Eilutei unikalumą suteikia vid ir pakid stulpelių pora.
+* Vaistinio preparato stadijos (Informacija apie vaistinio preparato stadijų pokyčius laike) - vieną eilutę sudaro vaistinis preparatas ir informacija apie stadijos pasikeitimą. Eilutei unikalumą suteikia vid, stadija ir stadija_sukurta stulpelių kombinacija.
+Abi lenteles galima sujungti per vid stulpelius.
+Dėl techninių klausimų ar galimų klaidų kreiptis į atverimas@stat.gov.lt"""
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues,,
+,,,,,,,,,,,,,,
+,,,,PreparataiPakuotes,,,vda_id,,,4,,,Valstybinės vaistų kontrolės tarnybos duomenys apie vaistinius preparatus ir jų pakuotes,
+,,,,,vda_id,string,,,,4,open,,VDA ID,Valstybės Duomenų Agentūros įrašo identifikatorius.
+,,,,,vid,string,,,,4,open,,Vaisto ID,"""Unikalus vaistinio preparato identifikatorius, automatiškai generuojamas sistemos."""
+,,,,,pak_id,string,,,,4,open,,Pakuotės ID,"""Unikalus vaistinio preparato pakuotės identifikatorius, automatiškai generuojamas sistemos, kiekvienai naujai vaistinio preparato pakuotei didinamas vienetu."""
+,,,,,npak_id,string,,,,4,open,,Nacionalinis pakuotės ID,"""Unikalus nacionalinis vaistinio preparato pakuotės identifikatorius, automatiškai generuojamas sistemos, kiekvienai naujai vaistinio preparato pakuotei didinamas vienetu."""
+,,,,,npak7_id,string,,,,4,open,,Septynių skaitmenų pakuotės ID,"""Unikalus nacionalinis septynženklis vaistinio preparato pakuotės identifikatorius, automatiškai generuojamas sistemos: prie npak_id pridedamas skaičius 1000000."""
+,,,,,atc_kodas,string,,,,4,open,,ATC kodas,Veikliosios medžiagos anatominės-terapinės-cheminės klasifikacijos kodas.
+,,,,,preparato_pav,text,,,,2,open,,Vaistinio preparato pavadinimas,"""Vaistinio preparato sugalvotas pavadinimas, vietoje formalaus ilgo pavadinimo."""
```

Comment:
text tipo dar nėra, reikėtų string nurodyt, ir pridėti komentarą (pridedu pavyzdį, bet be kablelių reikiamo skaičiaus - "comment,type,,"update(property: ""grupes_pavadinimas@lt"", type: ""text"")",4,open,spinta:204").
Stulpelio pavadinimas su underscore kalba nurodyta, pavyzdžiui veiklioji_medz_lt.